lib/datepicker.js

```diff
@@ -219,7 +220,8 @@ var DatePicker = _react2.default.createClass({
       timePickerButton: this.props.timeDisabled ? false : this.props.timePickerButton,
       onToggle: this.handleToggleTime,
       showTimePicker: this.state.showTimePicker,
-      onRemoveTime: this.handleRemoveTime });
+      onRemoveTime: this.handleRemoveTime,
+      fixedHeight: this.props.fixedHeight });
```

Comment:
@emilycw It seems like you added this last year but it's never generated into lib? https://github.com/flexport/react-datepicker/commit/a42c8503bd1d4e2f8d212e86ee587aed64565ff3